addons/binding/org.openhab.binding.omnilink/ESH-INF/thing/unit.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="omnilink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- These are all objects that are types of Units -->
+	
+	<thing-type id="unit">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="controller" />
+        </supported-bridge-type-refs>
+        <label>Unit</label>
+        <description>A basic unit object</description>
+        <channels>
+            <channel id="status" typeId="status" />
+        </channels>
+        <properties>
+            <property name="name" />
+            <property name="area" />
+            <property name="room" />
+        </properties>
+        <config-description>
+            <parameter name="number" type="integer" required="true">
+                <label>Unit Number</label>
+                <description>The number for a unit thing.</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+    
+	<thing-type id="upb">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="controller" />
+		</supported-bridge-type-refs>
+		<label>UPB Unit</label>
+		<description>A UPB based unit</description>
+		<channels>
+			<channel id="lightlevel" typeId="lightlevel" />
+		</channels>
+        <properties>
+            <property name="name" />
+            <property name="area" />
+            <property name="room" />
+        </properties>
+		<config-description>
+			<parameter name="number" type="integer" required="true">
+				<label>Unit Number</label>
+				<description>Unit number of the dimmer.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	
+	<thing-type id="alc">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="controller" />
+        </supported-bridge-type-refs>
+        <label>ALC Unit</label>
+        <description>A ALC base device</description>
+        <channels>
+            <channel id="lightlevel" typeId="lightlevel" />
+        </channels>
+        <properties>
+            <property name="name" />
+            <property name="area" />
+        </properties>
+        <config-description>
+            <parameter name="number" type="integer" required="true">
+                <label>Unit Number</label>
+                <description>Unit number of the ALC device.</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+    
+    <thing-type id="plc">
```

Comment:
PLC is complicated, I'm tempted to take this out right now, do we know of anyone who has this equipment?